src/AppInstallerCLICore/Command.cpp

```diff
@@ -199,16 +204,21 @@ namespace AppInstaller::CLI
 
             if (hasArguments)
             {
-                infoOut << Resource::String::AvailableArguments << std::endl;
-
+                bool atLeastOne = false;

```

Comment:
>bool atLeastOne = false; [](start = 16, length = 24)

Shouldn't be needed if the above code takes Visibility into account properly (as it does not right now).